src/utils/arrayMove.ts

```diff
@@ -0,0 +1,35 @@
+export function arrayMove<T>(array: T[], moveIndex: number, toIndex: number) {
```

Comment:
* Move to https://github.com/react-component/field-form/blob/master/src/utils/valueUtil.ts
* And test case for this.